### ISSUE TITLE
The source code publishing

### ISSUE THREAD

**Issue by Leptopt1los** [h-RAT/Willy_Firmware_CC1101_ESP32_Display_Touchscreen#4]
the code of your firmware is based on the Official Flipper Firmware (or one of its forks), which is distributed under the [GNU GPL 3 license](https://github.com/flipperdevices/flipperzero-firmware/blob/dev/LICENSE). Will the source code of your firmware be published, and in what time frame is it planned to do so?

**Comment by TQMatvey** [comment]
indeed, if not published, we should just take this repo down :D

**Comment by h-RAT** [comment]
> the code of your firmware is based on the Official Flipper Firmware (or one of its forks), which is distributed under the [GNU GPL 3 license](https://github.com/flipperdevices/flipperzero-firmware/blob/dev/LICENSE). Will the source code of your firmware be published, and in what time frame is it planned to do so?

No, my code is not based on Flipper Zero, ideas yes maybe but no code. Just an arduino alternative with public libraries and modules.

I was already working on something similar two years ago.

If you want to discuss it, you can contact me.

**Comment by h-RAT** [comment]
> 



> 

How to contact me? Maybe start by reading the github.

Yes, everything is implemented by myself, with libraries available long before Flipper Zero existed.

Yes the list looks like it, and? It is not the creator of the plugin who invented CAME bruteforce.

No flipper code was used, everything is much simpler with the libraries and everything existed long before Flipper Zero so I didn't even have to look the source of anything.

Didn't need Flipper Zero to use a CC1101 module lol.

I just use the word "Flipper Zero" for referencing and because my device is also capable of playing Flipper Zero .sub files, but I don't care about Flipper otherwise.

I will edit the github to indicate and thank the libraries used.

**Comment by Leptopt1los** [comment]
Good. If all the code was written by you without using flipper firmware, please explain:
1) where did you get information about the Kinggates protocol?
2) where did you get information about the BETT protocol? there is no such abbreviation and it was invented by the authors of flipper firmware for the protocol, which was reversed by them
3) where did you get information about Came 18 and 25 bit? there are no such versions, it's just an add-on. can you show the source where you found the information about it? 

the description of your protocols includes many things unique to flipper zero, information about which is not available in open sources

**Comment by h-RAT** [comment]
I just used my Flipper Zero to send the protocols and I used my ESP32 to analyze and understand them, so I could adapt them for ESP32. 

Anyway, I just think it's cool to be able to send almost any .sub file compatible with Flipper Zero, I don't think I'm annoying people lol.

If the problem is these two protocols, well I will remove and you would be happy.
